lib/pago.rb

```diff
@@ -0,0 +1,31 @@
+require 'ostruct'
+
+class Pago
+  class << self
+    ACCEPTED_PAYMENTS = [
+      :check,
+      :credit_card,
+      :po,
+    ]
+
+    def make_payment(order_id:, payment_method:, payment_details:)
+      raise "Unknown payment_method #{payment_method}" unless ACCEPTED_PAYMENTS.include?(payment_method)
+
+      cleaned_details = payment_details.map do |key, value|
```

Comment:
Clever approach. I imagine there's an easier way to do this same thing, but nothing is really jumping to mind. Probably some mix of regex and string manipulation. What you've done here is fine though.